radix-engine-interface/src/types/blueprint.rs

```diff
@@ -8,7 +8,7 @@ use sbor::rust::string::ToString;
 pub struct ObjectInfo {
     pub blueprint: Blueprint,
     pub global: bool,
-    pub type_parent: Option<GlobalAddress>,
+    pub blueprint_parent: Option<GlobalAddress>,
```

Comment:
I'd suggest we use `inner`/`outer` (and reserve `child`/`parent` for ownership).

So rename to `outter_object`?

Reply:
cool sounds good